core/services/feeds/service.go

```diff
@@ -362,8 +362,8 @@ func (s *service) ListManagersByIDs(ctx context.Context, ids []int64) ([]FeedsMa
 		return nil, errors.Wrap(err, "failed to list managers by IDs")
 	}
 
-	for _, manager := range managers {
-		manager.IsConnectionActive = s.connMgr.IsConnected(manager.ID)
+	for i, manager := range managers {
+		managers[i].IsConnectionActive = s.connMgr.IsConnected(manager.ID)
```

Comment:
isnt this the same behaviour as before?

Reply:
before it raised a warning for an ineffective assign. I think the values aren't pointers so it was updating the `manager` value inside the loop rather than the actual value in the array